index.html

```diff
@@ -29,352 +32,374 @@
+            </div>
           </div>
-          <div class="base-button flex-card">
-            <a href="">See Project</a>
+          <!---End of desktop card 1 Mark up-->
+          <div class="s1-container">
+            <div class="image-container"></div>
+            <div class="content">
+              <h2>Tonic</h2>
+              <div class="list-container">
+                <ul>
+                  <li><a href="" class="text">CANOPY</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-2">Back End Dev</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-3">2015</a></li>
+                </ul>
+              </div>
+              <div class="para">
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+              </div>
+
+              <div class="base-links">
+                <ul>
+                  <li><a href="" class="base-link-text1">html</a></li>
+                  <li><a href="" class="base-link-text2">css</a></li>
+                  <li><a href="" class="base-link-text3">javascript</a></li>
+                </ul>
+              </div>
+              <div class="base-button">
+                <a href="">See Project</a>
+              </div>
+            </div>
           </div>
         </div>
-      </div>
-      <div id="section-3">
-        <!---Begin Markup for Desktop card 3-->
-        <div class = "s3-container-dt">
-          <div class = "img-container3-dt">
-   <img src = "images/desktop-section-3.png" alt = "The book"/>  
-              </div>
-              <div class = "card-text">
-          <h2>FaceBook 360</h2>
-          <div class="list-container3-dt">
-            <ul>
-              <li><a href="" class="text">FACEBOOK</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-2">Full Stack Dev</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-3">2015</a></li>
-            </ul>
-          </div>
-          <div>
-            <p>
-              Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
-            </p>
+        <div id="section-2">
+          <!---start markup for s2 Desktop-->
+          <div class="s2-container-dt">
+            <div class="card-text-2">
+              <h3>Multi-Post-Stories</h3>
+              <div class="list-container2-dt">
+                <ul>
+                  <li><a href="" class="text">FACEBOOK</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-2">Full Stack Dev</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
```

Comment:
- [ ]  I noticed here the alt is empty,, kindly add a text describing the image for accessibility issues 🙏🏻